MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -388,6 +388,35 @@ public void UpdateServerTimeOffset(long currentServerTime)
 			AnimationManager.UpdateServerTimeOffset(currentServerTime);
 		}
 
+		public void DeclarePreallocatedActors(GameObject[] objects, string guidSeed)
```

Comment:
Will every client will call this when they load a space? If so, that could be a lot of redundant traffic. Perhaps delay the execution of this until after synchronization and skip actors already present at that time.

Reply:
Yes, each client will call this. We need to in order to register these gameobjects as actors locally, which we need for patching. You could make the case for not sending object-spawned, but that's not how the other actor creation flows work, and I'd like to keep it consistent.

Reply:
I have a concern over cross-platform support. This won't work for MREs that want to run across multiple platforms (Unreal & Altspace, for example). On the other hand, this does unlock a lot of capability for MREs authored just for Altspace. If we want to include this in the runtime API, we should be very clear about that.